src/sympc/api.py

```diff
@@ -197,7 +200,15 @@
     ("sympc.protocol.spdz.spdz.div_wraps", "sympc.tensor.ShareTensor"),
     (
         "sympc.session.Session.przs_generate_random_share",
-        "sympc.tensor.ShareTensor",
+        sy.lib.misc.union.UnionGenerator[
```

Comment:
Q: There is no other solution than using the `UnionGenerator`? (there is not something like `Union`?)